docs/platform/howto/create-custom-cloud.rst

```diff
@@ -0,0 +1,47 @@
+Create a custom cloud in Aiven |beta|
+=====================================
+
+Learn how to add a custom cloud to your Aiven administrator's account and check out how it works.
+
+About creating a custom cloud
+-----------------------------
+
+You can create a custom cloud in your Aiven administrator's account using `Aiven Console <https://console.aiven.io/>`_.
+
+Limitations
+'''''''''''
+
+
+Prerequisites
+-------------
+
+* Aiven administrator's account
+* Access to `Aiven Console <https://console.aiven.io/>`_
+
+Create a custom cloud
+---------------------
+
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ as an administrator.
+2. From the left sidebar, select **Bring you own account**.
+3. In the **Bring you own cloud** view, select **Create custom cloud**.
+4. In the **Create custom cloud** wizard
```

Comment:
It might be helpful to describe the steps in the workflow, and organize the detail steps according to the high-level workflow.